bin/Powheg/production/Run3/13p6TeV/bb_H_2HDM_13p6TeV/bbH_mssm_template/bbH.input-base

```diff
@@ -0,0 +1,86 @@
+numevts NEVENTS
+
+! Production parameters for ttH production 
+hmass XHMASSXd0      ! mass of the HIggs boson [GeV]
+hwidth XHWIDTHXd0  ! width of the Higgs boson [GeV]
+hdamp XHDAMPXd0     ! h parameter for hdamp function: 1/4 (mH+2mb)
+
+hdecaymode -1   ! -1 no decay                
+                ! 0 all decay channels open
+                ! 1-6 d dbar, u ubar,..., t tbar 
+                ! 7-9 e+ e-, mu+ mu-, tau+ tau-
+                ! 10  W+W-
+                ! 11  ZZ
+                ! 12  gamma gamma     
+
+#pythiatune 320	! PYTHIA tune
+
+ih1   1           ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2   1           ! hadron 2 (1 for protons, -1 for antiprotons)
+ebeam1 6800d0     ! energy of beam 1
+ebeam2 6800d0     ! energy of beam 2
+
+! To be set only if using 'native' pdfs
+!ndns1 131         ! pdf set for hadron 1 (mlm numbering)
+!ndns2 131         ! pdf set for hadron 2 (mlm numbering)
+! To be set only if using LHA pdfs
+! 10050 cteq6m
+! 10042 cteq6l with LO alphas (corr. to vbfnlo LO set)
+! 10041 cteq6l with NLO alphas
+! 21000 MSTW2008 LO
+! 21100 MSTW2008 NLO
+! 23300 MSTW2008 NLO, NF=4
+! 11082 CT10 Nf4
+! 92000 PDF4LHC15 NLO, NF=4 
+
+! lhans1  325500      ! pdf set for hadron 1 (LHA numbering) origin 260000
+! lhans2  325500      ! pdf set for hadron 2 (LHA numbering) origin 260000
+
+lhans1  92000      ! pdf set for hadron 1 (LHA numbering) origin 260000
+lhans2  92000      ! pdf set for hadron 2 (LHA numbering) origin 260000
```

Comment:
what's the motivation to use this pdf set instead of the 325500

Reply:
the use of PDF4LHC15 is the recommendation of the bbH/bH subgroup of the LHCHWG
(see here: https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCHWGBBH#Recommendations_for_the_acceptan).